api/backend.proto

```diff
@@ -67,21 +67,20 @@ message FunctionConfig {
 }
 
 message FetchMatchesRequest {
-  // FunctionConfig specifies a MMF address and client type for Backend to establish connections with the MMF
+  // A config that specifies where the MatchFunction server is for this FetchMatches call.
   FunctionConfig config = 1;
 
-  // MatchProfiles that will be sent to thhe MMF specified in the FunctionConfig.
+  // A MatchProfile that describe what the returned Matches looks like for this FetchMatches call.
```

Comment:
Given MatchProfile is actually described in detail in the messages.proto, this can simply be:

MatchProfile sent to the MatchFunction invoked for this FetchMatches call